runtime/syntax/sml.vim

```diff
@@ -4,12 +4,18 @@
 " Maintainers:  Markus Mottl            <markus.mottl@gmail.com>
 "               Fabrizio Zeno Cornelli  <zeno@filibusta.crema.unimi.it>
```

Comment:
Last year, when I sent an email to this maintainer's email address, it was undeliverable.

Reply:
My email should still work, I don't know about Fabrizio's current address.